src/com/amazon/ion/benchmark/WriteRandomIonValues.java

```diff
@@ -52,7 +53,7 @@
     final static private List<Integer> NO_EXPONENT_VALUE_RANGE = null;
     final static private List<Integer> NO_COEFFICIENT_DIGIT_RANGE = null;
     final static private String NO_TIMESTAMP_TEMPLATE = null;
-    final static private IonStruct NO_CONSTRAINT_STRUCT = null;
+    final static public IonStruct NO_CONSTRAINT_STRUCT = null;
```

Comment:
> Can we please add a comment around usage of `NO_CONSTRAINT_STRUCT` constant?

Will add the general usage explanation for these constant. However, these constant will be considered removed because of the refactoring in the future PR.